piplmesh/account/views.py

```
@@ -75,4 +79,49 @@ def logout(request):
         url = request.POST.get(auth.REDIRECT_FIELD_NAME)
         return auth_views.logout_then_login(request, url)
     else:
-        raise exceptions.PermissionDenied
\ No newline at end of file
+        raise exceptions.PermissionDenied
+
+class PushView(generic_views.TemplateView):
```

If I want to raise an error on exception in views.py - what is the way to do it?
I am doing it like this `return debug.technical_500_response(request, *sys.exc_info())`.